modules/apps/transfer/keeper/relay.go

```diff
@@ -229,6 +238,15 @@ func (k Keeper) OnRecvPacket(ctx sdk.Context, packet channeltypes.Packet, data t
 			return sdkerrors.Wrap(err, "unable to unescrow tokens, this may be caused by a malicious counterparty module or a bug: please open an issue on counterparty module")
 		}
 
+		if denomTrace.GetPath() == "" {
```

Comment:
I think we actually want to do this for all tokens being sent/received https://github.com/cosmos/ibc-go/issues/2664#issuecomment-1439845851

Reply:
Let's check with @ValarDragon and @womensrights to agree on the requirement. :)

Reply:
Yes I think that this would be useful for all ibc assets, not just native assets. I followed up with Cosmostation and it seems like it could simplify generating [data like this](https://www.mintscan.io/osmosis/assets)

Reply:
Great, thanks everyone for weighing in. We'll make the necessary changes to support this then.

Reply:
@crodriguezvega what do you think about making a follow up PR that supports ibc assets and merge this one? I can lead the course of the the follow up PR

Reply:
@stackman27 I think I prefer to continue working on this PR, so that the feature is complete when it gets merged. I started doing some changes this morning. I can push those now and you can pick up from there, if you want. I was starting to adjust the tests that I added.

Reply:
Ok, just pushed my changes. The test should be failing now.

Reply:
perfect! i will pick it up from here and ask you questions if i get stuck